aptos/example/mint_nft/README.md

```diff
@@ -0,0 +1,19 @@
+## 100$ USD Mint
+
```

Comment:
Could we rename the `example` directory to `examples`? As there are now multiple examples.

Reply:
Done